css/CSS2/visufx/overflow-applies-to-001-ref.xht

```diff
@@ -35,13 +36,17 @@
      </div>
 
      <div id="child3">
-       <img id="grandchild3" src="../support/1x1-green.png" alt="green" />
+       <img id="grandchild3" src="support/1x1-green.png" alt="green" />
```

Comment:
It seems like the old path works and the new path doesn't... or am I missing something?

Reply:
I expect the old path doesn't work in the built output and the new path doesn't work in the unbuilt input. Probably need *another* copy of those files in `support`. :/